src/category_theory/endofunctor/algebra.lean

```diff
@@ -0,0 +1,201 @@
+/-
+Copyright (c) 2019 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison, Bhavik Mehta, Johan Commelin, Reid Barton, Rob Lewis, Joseph Hua
```

Comment:
This seems to be a new file. Did all these authors contribute? I don't think I did. Should this be `2019 Scott Morrison`?

Reply:
this is basically a copy of (some of) `category_theory.monad.algebra` so i was told to look at the git blame of that and name those involved.

Reply:
at the same time though, I will not be removing `monad.algebra` because I think it would be more convenient to keep everything there and have a forgetful functor from monad algebras into endofunctor algebras

Reply:
Ok, so you can copy the list of authors. But I do think you should change the copyright line to `2022 Your Name`.